agents/README.md

```diff
@@ -0,0 +1,223 @@
+# Reinforcement Learning with [tensorflow/agents](https://github.com/tensorflow/agents)
+
+[![Docker Repository on Quay](https://quay.io/repository/cwbeitel/agents-demo/status "Docker Repository on Quay")](https://quay.io/repository/cwbeitel/agents-demo)
+
+Here we provide a demonstration of training a reinforcement learning agent to perform a robotic grasping task using Kubeflow running on Google Kubernetes Engine. In this demonstration you will learn how to paramaeterize a training job, submit it to run on your cluster, monitor the job including launching a tensorboard instance, and finally producing renders of the agent performing the robotic grasping task.
+
+For clarity and fun you can check out what the product of this tutorial will look like by clicking through the render screenshot below to a short video of a trained agent performing a simulated robotic block grasping task:
+
+[![](demo/render_preview.png)](https://youtu.be/0X0w5XOtcHw)
+
+Below are instructions to first set up various preliminaries then launch the example notebook in a variety of forms.
+
+### Setup
+
+##### GCP and Kubeflow configuration
+
+First we will need to ensure that the following Google Cloud Platform resources are configured correctly:
+1. A bucket exists to which to write training job logs.
+2. Kubeflow has been deployed on Google Kubernetes Engine
+3. Service account credentials exist within that cluster to permit TensorBoard to access Google Cloud Storage.
+4. That cluster has a namespace "rl" within which we will be running our training jobs.
+
+###### Creating a logs bucket
+
+Credentials are needed in order to make the calls to the Google Cloud platform APIs dispatched in subsequent steps. You can start by authenticating with GCloud in the usual way (i.e. `gcloud auth login` followed by `gcloud config set project <your-project-name>`).
+
+First, we need to create a Google Cloud Storage bucket to store job logs as well as a unique subdirectory of that bucket to store logs for this particular run. We can create that as follows:
+
+```bash
+gsutil mb -p <your-project-name> gs://<your-project-name>
+```
+
+Note that you'll need to set the path to this bucket at the beginning of the [demo notebook](demo/demo.ipynb) so remember it for later!
+
+###### Deploying Kubernetes
```

Comment:
Do we really need this section? The previous sections discussed deploying Kubeflow. Doesn't that imply you already have a Kubernetes cluster?

Reply:
Right. A short summary of assumptions and references to docs should be sufficient.